config/suppressions.xml

```diff
@@ -8,7 +8,6 @@
   <!-- can't split long messages between lines -->
   <suppress id="lineLengthXml" files="google_checks\.xml" lines="56,126"/>
   <!-- don't validate generated files -->
-  <suppress id="lineLengthXml" files="releasenotes\.xml"/>
```

Comment:
@romani please confirm this suppression shouldn't stay as it may come back and needed in the future.

Reply:
I manually update new release notes to match 100 char limit.

if we do not need this any more, and all other is already updated - awesome. Lets not have this suppression.